specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -27,6 +27,7 @@ useful for a wide audience.
 
 * VDK: https://github.com/vmware/versatile-data-kit/wiki/dictionary#vdk
 * Notebook: A .ipynb file.
+* Notebook cell: a multiline text input field
```

Comment:
For external terms. Link to the defintion. This seems to have come from https://jupyter-notebook.readthedocs.io/en/stable/notebook.html#structure-of-a-notebook-document 
So add a link 
```
Notebook cell: a multiline text input field ([defined in details here](https://jupyter-notebook.readthedocs.io/en/stable/notebook.html#structure-of-a-notebook-document))
```

Reply:
I will add it with the next change on VEP